R/tab_create_modify.R

```diff
@@ -162,6 +233,93 @@ tab_spanner <- function(data,
   data
 }
 
+resolve_spanner_level <- function(
```

Comment:
`resolve_spanner_level` should only take `column_names`; resolve them ahead of time.

Reply:
Now done.